config/config.exs

```diff
@@ -68,10 +68,9 @@ config :phoenix, :json_library, Jason
 config :geo_postgis,
   json_library: Jason
 
-config :bike_brigade, BikeBrigade.Tasks.MailchimpImporter,
-  list_id: {:system, "MAILCHIMP_LIST_ID"}
+config :bike_brigade, BikeBrigade.Tasks.MailchimpImporter, list_id: {:system, "MAILCHIMP_LIST_ID"}
```

Comment:
I think some of these changes are from the command run for #396 maybe not including the `config` dir? But not quite sure why stuff in `lib` is changed so much.

Reply:
I think the stuff in lib changed because mix format ran without the `:ecto` import dep - which we have in `.formatter.exs` in the main dir -- i wonder if you ran mix format from within the subdirectory and the bug you found below triggered.

Do you mind trying to undo all your changes in lib and running the formatter again - on my system it doens't add all the parenthesis which is what I want.

The idea is that you generally parenthesize, but the ecto dsl skips them which is what I'm hoping the formatter to enforce.

Reply:
👍 that seems to have removed almost all changes from `lib`, thanks!